core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -803,6 +803,27 @@ object ConsumerGroupCommand extends Logging {
       rows.foldRight("")(_ + "\n" + _)
     }
 
+    override def deleteGroups(): Map[String, Errors] = {
+      val groupsToDelete = opts.options.valuesOf(opts.groupOpt).asScala.toList
```

Comment:
Hmm.. It's a little weird that we allow multiple groups to be passed when using the new consumer, but we expect a single group for the old consumer. If we're to stay consistent, do you think it would be restrictive in practice to only support deletion of a single group at a time?

Reply:
In the meantime I'll look at your other feedback (thanks btw) regarding this one, it seems the old consumer also supports deleting multiple groups, i.e. `... --delete --group group1 --group group2` works and attempts to remove both groups.

I originally wanted to support single group deletion only, but after considering the existing behavior for old consumer decided otherwise.

Reply:
Ah, you are right. The name `deleteForGroup` is kind of misleading. Maybe it just needs to be pluralized. I wouldn't hate it if we came up with better names for all of these `deleteForXXX` APIs.

Reply:
Sure, I gave this a quick try. Let me know if you have better suggestions.